metadata-service/restli-client/src/main/java/com/linkedin/entity/client/EntityClient.java

```diff
@@ -295,7 +295,7 @@ default String wrappedIngestProposal(@Nonnull MetadataChangeProposal metadataCha
       @Nonnull final Authentication authentication, final boolean async) {
     try {
       return ingestProposal(metadataChangeProposal, authentication, async);
-    } catch (RemoteInvocationException e) {
+    } catch (Exception e) {
```

Comment:
What are the additional exceptions we're handling here? Doesn't look like the ingestProposal method definition changed to add any

Reply:
Good catch, was messing with a variation here that should be reverted.